autospec/config.py

```diff
@@ -831,8 +831,10 @@ def write_default_conf_file(name, description):
     # Parse the version-specific patch lists
     for version in versions:
         verpatches[version] = read_conf_file(os.path.join(path, '.'.join(['series', version])))
+        if any(p.lower().startswith('cve-') for p in verpatches[version]):
+            config_opts['security_sensitive'] = True
+            rewrite_config_opts(path)
```

Comment:
Would you mind just setting a flag for rewrite_config to run outside the loop?

Reply:
Done